### ISSUE TITLE
Initialization weight of PolarFormer-Vov.

### ISSUE THREAD

**Issue by Daniel-xsy** [fudan-zvg/PolarFormer#8]
Sorry to bother you again. Can you please provide the initialization weight of the PolarFormer-Vov since it's initialized from [DD3D](https://github.com/TRI-ML/dd3d) as mentioned in the paper? I did not find it in this codebase. Thanks so much!

**Comment by yanqinJiang** [comment]
Hi, the vovnet-pretrained model is downloaded from [DETR3D repo](https://drive.google.com/drive/folders/1h5bDg7Oh9hKvkFL-dRhu5-ahrEp2lRNN), named dd3d_det_final.pth there.

**Comment by Daniel-xsy** [comment]
@yanqinJiang Thanks for your information.

**Comment by Daniel-xsy** [comment]
Hi @yanqinJiang

I noticed in https://github.com/megvii-research/PETR/issues/75#issuecomment-1323666167, it mentioned that the `dd3d_det_final.pth` is first pretrained on DDAD15M dataset and further trained on **nuScenes trainval set** with DD3D. So I'm wondering if is it reasonable to evaluate this version on nuScenes validation set due to **information leakage** (nuScenes validation set is used during DD3D pre-train).

We are working on benchmarking ood robustness of camera-only detectors, and PolarFormer-Vov shows superior performance. 
I'm wondering if it is caused by the reason mentioned above. If you are interested in the results, please kindly refer to [RoboDet](https://github.com/Daniel-xsy/RoboDet).

**Comment by Daniel-xsy** [comment]
The results shown in this codebase of `PolarFormer-Vov` on nuScenes validation set also largely surpass the ResNet version.

**Comment by yanqinJiang** [comment]
I didn't notice that `dd3d_det_final.pth`  was trained on **nuScenes trainval set**. Thanks for pointing out my mistake. Of course, it is unfair to evaluate PolarFormer-Vov initialized from `dd3d_det_final.pth` on validation set due to information leak. I will delete this result. Thanks again. For fair comparison, you can train a PolarFormer-Vov initialized from `fcos3d_vovnet_imgbackbone-remapped.pth` as mentioned in PETR repo.

**Comment by Daniel-xsy** [comment]
Thanks for the update.
